hack/e2e.sh

```diff
@@ -102,5 +116,24 @@ while [ $ATTEMPTS -ge 0 ]; do
   fi
 done
 
+# Wait for cincinnati metrics to be recorded
+# Find out the token Prometheus uses from its serviceaccount secrets
+# and use it to query for GB build info
+PROM_ENDPOINT=$(oc -n openshift-monitoring get route thanos-querier -o jsonpath="{.spec.host}")
+PROM_TOKEN=$(oc -n openshift-monitoring get secret \
+  $(oc -n openshift-monitoring get serviceaccount prometheus-k8s \
+    -o jsonpath='{range .secrets[*]}{.name}{"\n"}{end}' | grep prometheus-k8s-token) \
+  -o jsonpath='{.data.token}' \
+  -o go-template='{{.data.token | base64decode}}')
+
+ATTEMPTS=10
+DELAY=30
+while [ $ATTEMPTS -ge 0 ]; do
+  curl -kLs -H "Authorization: Bearer ${PROM_TOKEN}" https://${PROM_ENDPOINT}/api/v1/query\?query\=cincinnati_gb_build_info | grep "metric" && break
```

Comment:
nit: `"https://${PROM_ENDPOINT}/api/v1/query?query=cincinnati_gb_build_info"` to avoid the backslashes and protect against the unlikely case that `PROM_ENDPOINT` has some shell-sensitive characters.

Also no need for the quotes in `"metric"`.

Reply:
>Also no need for the quotes in "metric".

I added this intentionally, so that it would be highlighted in most editor and the command would make a bit more sense :)